models/app.go

```diff
@@ -23,6 +23,11 @@ type AppListItem struct {
 	// example: true
 	IstioSidecar bool `json:"istioSidecar"`
 
+	// Define if belongs to a namespace labeled as ambient
```

Comment:
I see in another place where "IstioAmbient" is defined as the pod having an ambient annotation (see below from pod.go in this file) but this comment says this flag is true if the namespace is labeled (no mention of pod annotations). Please double check and just make sure this comment is correct and this really only involves namespace labels. This may be correct already - it could be different semantics between app and workload; I just want to make sure.

Reply:
Yes, you are right. The check is done by looking at the pod and the ambient annotation, so I've updated the comment.